messaging/src/test/java/org/axonframework/utils/InMemoryStreamableEventSource.java

```diff
@@ -57,15 +57,15 @@ public class InMemoryStreamableEventSource implements StreamableMessageSource<Tr
     private Runnable onAvailableCallback = null;
 
     /**
-     * Construct a default {@link InMemoryStreamableEventSource}. If stream callbacks should be supported for testing,
-     * use {@link InMemoryStreamableEventSource#InMemoryStreamableEventSource(boolean)}.
+     * Construct a default {@link AsyncInMemoryStreamableEventSource}. If stream callbacks should be supported for testing,
+     * use {@link AsyncInMemoryStreamableEventSource#AsyncInMemoryStreamableEventSource(boolean)}.
```

Comment:
I do not think that the JavaDoc changes in this file are correct. Both constructors still make a `InMemoryStreamableEventSource`, not an `AsyncInMemoryStreamableEventSource`. Or did you intend to remove **this** class, and did git/IDE make some weird assumptions?